examples/mnit_ballz/src/forces.nit

```diff
@@ -0,0 +1,34 @@
+# This module contains all the forces applicables to the objects in the game
```

Comment:
The "This module contains" part is useless. Maybe "Physical model for the applicable forces to the objects in the game"